ibis/expr/types/relations.py

```diff
@@ -1177,6 +1181,30 @@ def prevent_rewrite(self, client=None) -> TableExpr:
     def materialize(self) -> TableExpr:
         return self
 
+    def alias(self, alias: str) -> ir.TableExpr:
```

Comment:
I find it a little confusing, could't we call it `view`?

Reply:
`view` is already a method that creates a referentially distinct alias to a relation, and is unrelated to a temporary name.